src/main/java/org/embulk/output/SnowflakeOutputPlugin.java

```diff
@@ -44,6 +46,10 @@ public interface SnowflakePluginTask extends PluginTask {
     @ConfigDefault("\"\"")
     String getPrivateKey();
 
+    @Config("privateKeyPassphrase")
```

Comment:
It seems better to convert this property to snake_case. While privateKey is in camelCase, all other properties are in snake_case, so it would be best to unify them under that convention. I was the one who added privateKey, but after merging it into the master branch, I realized the naming mistake. It has remained unchanged up to this point

Reply:
Sure! I've updated the names to snake_case. Please be aware that this change may introduce some backward compatibility issues.

Reply:
Thank you for the modification! My apologies, but as it would break backward compatibility, there's no need to change the privateKey.

Reply:
I am considering addressing the snake_case conversion of privateKey in the future.